helper/resource/testing.go

```diff
@@ -451,49 +447,6 @@ type TestStep struct {
 	providers map[string]*schema.Provider
 }
 
-// Set to a file mask in sprintf format where %s is test name
-const envLogPathMask = "TF_LOG_PATH_MASK"
```

Comment:
I believe `TF_LOG_PATH_MASK` may be used in our acctests in TeamCity, is this accounted for in the new implementation?

Reply:
I am investigating if this is something we can live without